src/email-template/templates/daily-message.template.ts

```diff
@@ -1,92 +1,102 @@
 
 @Injectable()
 export class DailyMessageEmailTemplate implements IEmailTemplate {
-    constructor(
-        private pointEventService: PointEventService
-        ) {
-    }
-
-    getName() {
-        return 'daily-message'
-    }
-
-    async render(user:User, {hourInUserTimeZone, minutesInUserTimeZone, lastSent, contextTimestamp}:{hourInUserTimeZone: number, minutesInUserTimeZone: number, lastSent: Date, contextTimestamp: Date}) {
-        const now = moment(contextTimestamp);
+  constructor(private pointEventService: PointEventService) {}
 
-        // First time sending daily message
-        if(!lastSent) {
-            const userRegistered = moment(user.createdAt);
-            const durationSinceRegistered = moment.duration(now.diff(userRegistered));
+  getName() {
+    return 'daily-message';
+  }
 
-            // Don't send a daily message within N hours of registration
-            if(false && durationSinceRegistered.asHours() < parseInt(process.env.HOURS_UNTIL_FIRST_DAILY_EMAIL)) {
-                console.log("Too soon after registration to send daily msg")
-                return false;
-            }
-        } else {
-            // Already sent daily message before
-            const last = moment(lastSent);
-            const durationSinceLastEmail = moment.duration(now.diff(last));
-            // Don't send new emails if less than 23 hours has passed since last
-            // 23 hours because otherwise if last email was sent at 12:05 and we now run at 12:00
-            // the difference would be something like 23.995
-            if(durationSinceLastEmail.asHours() < 23) {
-                console.log("Already sent daily email less than 23 hours ago")
-                return false;
-            }
-        }
+  async render(
+    user: User,
+    {
+      hourInUserTimeZone,
+      lastSent,
+      contextTimestamp,
+    }: {
+      hourInUserTimeZone: number;
+      minutesInUserTimeZone: number;
+      lastSent: Date;
+      contextTimestamp: Date;
+    },
+  ) {
+    const now = moment(contextTimestamp);
 
-        const oneDayAgo=now.subtract(1,"day").toDate()
-        const events = await this.pointEventService.findAllForUser(user.userid);
-        const highestDay = 5;
-        const pointsEarnedToday = user.events
-            .filter((event)=>{ return event.timestamp > oneDayAgo })
-            .map((event) => event.points)
-            .reduce((previous, current) => previous + current, 0)
+    // First time sending daily message
+    if (!lastSent) {
+      const userRegistered = moment(user.createdAt);
+      const durationSinceRegistered = moment.duration(now.diff(userRegistered));
 
-        if (!(pointsEarnedToday > 0)) {
-            return false;
-        }
-        /*if(pointsEarnedToday > highestDay) {            
-            // Percentile of people
-        }*/
+      // Don't send a daily message within N hours of registration
+      if (
+        false &&
```

Comment:
is this false is on purpose or for testing. because of this and the one below, now every minute the mail is being sent!!